beacon-chain/state/state-native/state_trie.go

```diff
@@ -84,10 +84,14 @@ var capellaFields = append(
 )
 
 const (
-	phase0SharedFieldRefCount    = 10
-	altairSharedFieldRefCount    = 11
-	bellatrixSharedFieldRefCount = 12
-	capellaSharedFieldRefCount   = 14
+	phase0SharedFieldRefCount     = 5
```

Comment:
why is this 5 ?

Reply:
multi value slices are not a part of `sharedFieldReferences`

Reply:
They are for the old state type, no ?